lib/tasks/code_climate.rake

```diff
@@ -1,6 +1,6 @@
 namespace :code_climate do
```

Comment:
Do we still need this rake task?

Reply:
I am not sure. I would keep it in case we want to trigger an update any time other than the cron schedule but I am not sure about it

Reply:
I'm inclined to delete it, to avoid any possible confusions down the road. @santib what's your opinion on this?

Reply:
Deleted it. Thanks